_data/validator-libraries-modern.yml

```
@@ -275,17 +275,28 @@
   implementations:
     - name: Hyperjump JSV
       url: https://json-schema.hyperjump.io
-      date-draft: [2019-09, 2020-12]
+      date-draft: [2020-12, 2019-09]
+      draft: [7, 6, 4]
+      notes: Supports multiple schemas and multiple instances; client-side validation
+    - name: json-everything
+      url: https://json-everything.net
+      date-draft: [2020-12, 2019-09]
+      draft: [7, 6]
+      notes: Powered by JsonSchema.Net; server-side validation
+    - name: jsonschema.dev
+      url: https://jsonschema.dev
+      date-draft: [2019-09]
       draft: [7, 6, 4]
```

https://jsonschema.dev only supports draft-07

```suggestion
      draft: [7]
```

---

Officially, yes. But I think because it uses ajv under the hood, and an older version, that it supports those drafts when $schema is specified.

---

I checked before I posted the comment. It only works with draft-07. I know ajv can be configured for other drafts, but you only get the latest draft (draft-07 in this case) by default. You might be able to configure it to support more than one draft at a time and switch off of `$schema`, but I know for sure that you don't get it by default.

---

I believe that ajv only supports one version at a time and `$schema` just allows you change the meta-schema that is used to validate the schema which gives the illusion of supporting other drafts. Most of the time, that will work, but not always. For example, consider ajv configured for draft-07 and a schema that uses the `$schema` for draft-04. If that schema uses the `const` keyword, then the validator should ignore it as an unknown keyword. But, the `const` would not be ignored because it's really a draft-07 implementation masquerading as draft-04.

---

Oh wow, it ignores the `$schema` keyword?

IMO, *every* implementation should give an error if it sees a `$schema` that it doesn't recognize (either one of the official metaschemas, for a version that isn't supported, or a custom metaschema if it doesn't support those and the $vocabulary keyword). Why isn't that in the spec?

---

I believe it IS in the spec now. And ajv may do that now. But this is an old version.

---

@jdesrosiers I can't remember exactly how it works and it may or may not support multiple versions at once. I'd have to find out.

I'm the meantime, I don't want to hold up this PR. I'll follow up with another if needs be.